Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsParsingService.cs

```diff
@@ -29,5 +49,23 @@ public IBotCommand ParseRequestToCommand(Type commandType, DiscordRequest reques
             }
             return (IBotCommand)instance;
         }
+
+        private bool CustomTemplateIsValid(Match match, BotCommandTemplate template)
+        {
+            if(!match.Success)
+            {
+                return false;
+            }
+            var requiredProperties = template.Properties.Where(x => !x.IsOptional).ToList();
+            if (match.Groups.Count - 1 < requiredProperties.Count)
```

Comment:
nie lepiej <= ?

Reply:
"- 1" chyba jest czytelniejsze w tym przypadku
chodzi o to że zawsze jest grupa o nazwie "0" w której jest cały wynik i chcemy ją ignorować
ale pewnie <= jest wydajniejsze
hmm
trudny wybór